packages/ui-components/src/ExpandableText/ExpandableText.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import styles from "./ExpandableText.module.scss";
+import classNames from "classnames/bind";
+import LinesEllipsis from "react-lines-ellipsis";
+
+interface ExpandableTextProps {
+  text: string;
+  maxLine: number;
+}
+
+const cx = classNames.bind(styles);
+
+export function ExpandableText({
+  text,
+  maxLine,
+}: ExpandableTextProps): React.ReactElement {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  useEffect(() => {
```

Comment:
> but wouldn't that happen anyway on a re-render (since the default state is false)?

IIRC the default value is only applied when the component is mounted, so an effect is necessary here I think!